Makefile

```diff
@@ -169,21 +239,18 @@ rm -rf $$TMP_DIR ;\
 }
 endef
 
-OPERATOR_SDK = $(shell pwd)/bin/operator-sdk
-OPERATOR_SDK_VERSION = v1.22.0
-operator-sdk: ## Download operator-sdk locally if necessary.
-	./utils/install-operator-sdk.sh $(OPERATOR_SDK) $(OPERATOR_SDK_VERSION)
-
 .PHONY: bundle
-bundle: export IMAGE_TAG := $(IMAGE_TAG)
-bundle: export BUNDLE_VERSION := $(VERSION)
-bundle: manifests kustomize operator-sdk ## Generate bundle manifests and metadata, then validate generated files.
+bundle: $(OPM) $(YQ) manifests kustomize operator-sdk ## Generate bundle manifests and metadata, then validate generated files.
 	$(OPERATOR_SDK) generate kustomize manifests -q
+	# Set desired operator image
 	cd config/manager && $(KUSTOMIZE) edit set image controller=$(IMG)
-	envsubst \
-		< config/manifests/bases/limitador-operator.clusterserviceversion.template.yaml \
-		> config/manifests/bases/limitador-operator.clusterserviceversion.yaml
+	# Update CSV
+	V="limitador-operator.v$(VERSION)" $(YQ) eval '.metadata.name = strenv(V)' -i config/manifests/bases/limitador-operator.clusterserviceversion.yaml
+	V="$(VERSION)" $(YQ) eval '.spec.version = strenv(V)' -i config/manifests/bases/limitador-operator.clusterserviceversion.yaml
+	V="$(IMG)" $(YQ) eval '.metadata.annotations.containerImage = strenv(V)' -i config/manifests/bases/limitador-operator.clusterserviceversion.yaml
```

Comment:
Combining [Kustomize](https://kustomize.io) and [yq](https://mikefarah.gitbook.io/yq/) to build manifests from templates while setting custom values from env vars.

In [Authorino Operator](https://github.com/Kuadrant/authorino-operator/blob/8971551f9bc83b28bf029a0228242eb716a7be4b/Makefile#L97), we too mix [Kustomize](https://kustomize.io) with (in that case) [envsubst](https://www.gnu.org/software/gettext/manual/html_node/envsubst-Invocation.html) for similar purpose.

I wonder whether we shouldn't try to normalise these customisations to start relying on a single technology to solve it all. For example, I think Kustomize with a [custom transformer](https://github.com/kubernetes-sigs/kustomize/blob/master/examples/transformerconfigs/crd/README.md) and [`configMapGenerator `](https://github.com/kubernetes-sigs/kustomize/blob/master/examples/configGeneration.md) could do the job here. Not sure how much simpler or more readable it would be tho.

WDYT?

Reply:
I moved from envsubst to yq for two reasons:
* With envsubsts we need to manage *.template.yaml files
*  envsubsts works only with env vars, so `export FOO=BAR` was required

Reply:
I prefer `yq` over `envsubst` too, but I reckon it'd be better to craft our manifests and configs with just Kustomize will make it cleaner and remove the necessity of an extra tool. However, I'm not entirely sure if we could reach the same output and it would be better to leave it for another PR